beater/beater.go

```diff
@@ -47,16 +47,17 @@ func New(b *beat.Beat, ucfg *common.Config) (beat.Beater, error) {
 
 func (bt *beater) Run(b *beat.Beat) error {
 	var err error
+	logger := logp.NewLogger("beater")
```

Comment:
There are two identical calls to `logp.NewLogger("beater")`. I suggest we put this logger on the beater instead. WDYT?